test/functional/lua/json_spec.lua

```diff
@@ -0,0 +1,26 @@
+local helpers = require('test.functional.helpers')(after_each)
+local clear = helpers.clear
+local exec_lua = helpers.exec_lua
+local eq = helpers.eq
+
+describe('lua-cjson', function()
+  before_each(function()
+    clear()
+  end)
+
+  it('can encode json', function()
+
+    eq(
+     '{"hello":"world"}' ,
+      exec_lua([[return vim.json.encode({ hello = 'world'})]])
+    )
+  end)
+
+  it('can decode json', function()
+
+    eq(
+      { hello = 'world'},
+      exec_lua([[return vim.json.decode('{"hello":"world"}')]])
```

Comment:
i kinda assumed more tests were on the way. Maybe copy a few from `json_functions_spec.lua` ? 

Or parameterize (most of) the tests in `json_functions_spec.lua` so that it tests both.

Reply:
Looking through the tests, I think parameterizing them is going to be non-trivial, seems like many of them deal with vimscript specific behavior, or eval-ing a string representation of the vim command.